bootstrap/sponge/src/main/java/org/geysermc/platform/sponge/GeyserSpongePlugin.java

```diff
@@ -47,7 +47,7 @@
 import java.io.IOException;
 import java.util.UUID;
 
-@Plugin(id = "geyser", name = GeyserConnector.NAME + "-Sponge", version = GeyserConnector.VERSION, url = "https://geysermc.org", authors = "GeyserMC")
+@Plugin(id = "geyser", name = GeyserConnector.NAME + "-Sponge", version = GeyserConnector.VERSION_STATIC, url = "https://geysermc.org", authors = "GeyserMC")
```

Comment:
Wouldn't this just set the version directly to `git` and nothing more?

Reply:
This is only the one used internally and doesn't show in the main header. I did it because if I use the dynamic one it errors and prevents me from compiling so I thought that would be the best way.